src/Factories/ErrbitLoggerFactory.php

```diff
@@ -10,6 +10,32 @@
 use Psr\Log\LogLevel;
 
 class ErrbitLoggerFactory {
+
+	/**
+	 * Data from PayPal IPN we don't want to have in ErrBit
+	 *
+	 * Should be in sync with PayPalBookingTransformer,
+	 * but since that is an internal class, we repeat the values here.
+	 */
+	private const FILTERED_PPL_PARAMS = [
```

Comment:
I think this should be an allow list so we don't accidentally log personal data if PayPal add a new field or change a field name. Maybe overkill, but I think it's better to be safe than sorry.

Reply:
Changed, please re-review